uber/models.py

```diff
@@ -693,53 +693,6 @@ def shift_badges(self, badge_type, badge_num, *, until=None, **direction):
                 a.badge_num += shift
             return True
 
-        def change_badge(self, attendee, badge_type, badge_num=None):
```

Comment:
Was this function just not called anywhere? I grepped through the code and couldn't find any references.